### ISSUE TITLE
Does not provided named default

### ISSUE THREAD

**Issue by fredski02** [nuxt-community/nuxt-logrocket#342]
I followed the guide exactly from here https://nuxt.com/modules/logrocket

my **nuxt** version is **3.3.1**
my **nuxt-logrocket** version is **2.0.6**

**nuxt.config.ts**
```ts
  modules: [
    'nuxt-logrocket',
 ] 
 ...
  logRocket: {
    id: `NUXT_LOG_ROW`,
    dev: true
  },
```

In my browser, i get the following error

```
ugin.client.mjs:1 Uncaught SyntaxError: The requested module '/_nuxt/node_modules/.pnpm/logrocket@4.0.3/node_modules/logrocket/dist/build.umd.js?v=b14f0b08' does not provide an export named 'default' (at plugin.client.mjs:1:1)
```

**Comment by farzadso** [comment]
@Atinux Since you had the same issue on nuxt tailwind, I was wondering if you could point me in the right direction as I couldn't find any commits that fixed this issue.

**Comment by Atinux** [comment]
I guess @ineshbose could help

**Comment by camille500** [comment]
Having the same issue

**Comment by ineshbose** [comment]
@farzadso I tried to investigate this issue (coming from client plugin) but can't say for fix. Do you know what exact issue you are referring to from the Tailwind module? (I am unable to recall)

**Comment by farzadso** [comment]
@ineshbose The only relevant discussion I found was this : https://github.com/nuxt/nuxt/issues/13247

Hope it rings some bells.

**Comment by ineshbose** [comment]
> @ineshbose The only relevant discussion I found was this : https://github.com/nuxt/nuxt/issues/13247
> 
> Hope it rings some bells.

Oh this was because we weren't bundling a dependency with the module (was added to `unbuild.externals`) and we anyway switched dependency. So it wasn't related.. (let me continue going through the discussion and experimenting in Stackblitz)

**Comment by farzadso** [comment]
> > @ineshbose The only relevant discussion I found was this : [nuxt/nuxt#13247](https://github.com/nuxt/nuxt/issues/13247)
> > Hope it rings some bells.
> 
> Oh this was because we weren't bundling a dependency with the module (was added to `unbuild.externals`) and we anyway switched dependency. So it wasn't related.. (let me continue going through the discussion and experimenting in Stackblitz)

Yeah, I went through the commits and didn't find anything related.

Much appreciated.

**Comment by dolbex** [comment]
@farzadso any luck with this issue - we'd love to try working with Logrocket but cannot seem to get past this issue.

**Comment by Me-Phew** [comment]
Do you think [this issue](https://github.com/AlexLavoie42/Nuxt-Mapbox/issues/49) could be related?

**Comment by farzadso** [comment]
@Me-Phew I upgrade nuxt to 3.6.5 and the issue still exists.

**Comment by ineshbose** [comment]
@farzadso would we be able to add logrocket to imports like so:

```ts
addImports({
      name: 'default',
      as: 'logRocket',
      from: 'logrocket'
})
```

and then in the plugin use:

```ts
import { logRocket } from '#imports'
```

I tried it on the repo playground but probably not testing properly.

**Comment by farzadso** [comment]
@ineshbose Can you give me a link to your playground please?

**Comment by ineshbose** [comment]
Closed the tab... it was just the repo opened on Stackblitz with the above change!

**Comment by farzadso** [comment]
@ineshbose Even adding your proposed changes doesn't fix it. You can see the changes on `v2.0.9`.
